python/lsst/jointcal/jointcal.py

```diff
@@ -723,6 +723,7 @@ def _logChi2AndValidate(self, associations, fit, model, chi2Label="Model",
             raise ValueError("Model is not valid: check log messages for warnings.")
         if writeChi2Name is not None:
             fit.saveChi2Contributions(writeChi2Name+"{type}")
+            self.log.info("Wrote chi2 contributions files: %s"%writeChi2Name)
```

Comment:
If you use a comma instead of `%`, I think the logger will only do the string interpolation if the appropriate log level is enabled.  Doesn't matter for this case, but it's a good habit to get into.

Reply:
Thanks for the reminder. I'd forgotten about that.